fake/fake_clock_121.go

```diff
@@ -0,0 +1,35 @@
+//go:build go1.20
+
+package fake
+
+import (
+	"context"
+	"time"
+)
+
+func (f *Clock) ContextWithDeadlineCause(ctx context.Context, t time.Time, cause error) (context.Context, context.CancelFunc) {
```

Comment:
Since they're exported symbols, it would be nice to give these methods golint-compatible doc-comments. (especially since it's an external package).